packages/dd-trace/src/appsec/remote_config/capabilities.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
   APM_TRACING_LOGS_INJECTION: 1n << 13n,
   APM_TRACING_HTTP_HEADER_TAGS: 1n << 14n,
   APM_TRACING_CUSTOM_TAGS: 1n << 15n,
+  ASM_PROCESSOR_OVERRIDES: 1n << 16n,
+  ASM_CUSTOM_DATA_SCANNERS: 1n << 17n,
+  ASM_EXCLUSION_DATA: 1n << 18n,
```

Comment:
these capabilities do not exist in https://github.com/DataDog/dd-go/blob/prod/remote-config/pkg/rc/capabilities.go